TestPrograms/docs_examples/tour/README.md

```
@@ -0,0 +1,218 @@
+```wfl
+store name as "Alice"
+store age as 30
+store is_member as yes
+
+display "Name: " with name
+display "age is a " with typeof of age
+
+change age to age plus 1
+display "Next year: " with age
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)